pytket/binders/circuit/Circuit/add_classical_op.cpp

```diff
@@ -242,15 +242,19 @@ void init_circuit_add_classical_op(
             std::vector<bool> vals(reg.size());
             for (unsigned i = 0; i < reg.size(); i++) {
```

Comment:
As discussed I think we should raise an exception if `value` does not fit in the register, i.e. if
```
value >> reg.size() != 0
```

Reply:
Done.